### PR TITLE
Fix plugin test run.sh script parameter error

### PR DESCRIPTION
**Pull request by aderm**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [x] Bug fix
- [ ] New feature provided
- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

**Comment by wu-sheng** [comment]
@dmsolr Your review

File: test/plugin/run.sh [hunk 1]
```diff
@@ -113,8 +114,17 @@ do_cleanup() {
...
+check_scenario_name_param() {
+    if test -z "$scenario_name"; then
+        echo "Missing value for the scenario argument"
+        exit 0
+    fi
+}
+
 start_stamp=`date +%s`
 parse_commandline "$@"
+check_scenario_name_param()
```
Comment (dmsolr) [inline test/plugin/run.sh]: have you run it? This statement is a syntax error. And it must be space below L129.
I suggest:
`test -z "$scenario_name" && exitWithMessage "Missing value for the scenario argument"`

BTW, remove unnecessary empty lines, Please.

Comment (aderm) [reply test/plugin/run.sh]: o, () must be delete. copy error

Comment (aderm) [reply test/plugin/run.sh]: o, copy error. () must be delete

Comment (aderm) [reply test/plugin/run.sh]: @dmsolr please review

File: test/plugin/run.sh [hunk 2]
```diff
@@ -116,6 +117,8 @@ do_cleanup() {
 start_stamp=`date +%s`
 parse_commandline "$@"
 
+test -z "$scenario_name" && exitWithMessage "Missing value for the scenario argument"
+
 if [[ "$cleanup" == "on" ]]; then
     do_cleanup
     exit 0
```
Comment (dmsolr) [inline test/plugin/run.sh]: Line:120 must be behind Line:125

Comment (aderm) [reply test/plugin/run.sh]: done @dmsolr

**Comment by dmsolr** [comment]
LGTM

**Pull request merged**